### DISCUSSION TITLE
Option to return extra tags in the output

### DISCUSSION THREAD

**Discussion by xinyuluo** [komoot/photon#663] [Q&A] [answered]
Hi, is it possible for photon to have an option to pass an "extratags" paramater to return some extra information about the place?

Like for Nominatim, [https://nominatim.openstreetmap.org/lookup?osm_ids=W50637691&format=json&extratags=1](https://nominatim.openstreetmap.org/lookup?osm_ids=W50637691&format=json&extratags=1)

Please correct me if this is already implemented and I just didn't find the correct parameter to use.

Thank you!

**Comment by mtmail** [comment, marked as answer]
You can add extra tags data in a custom import (your own installation), they're not imported on [photon](https://photon.komoot.io/). See https://github.com/komoot/photon/issues/395#issuecomment-839146066

**Reply by xinyuluo** [reply]
Thank you @mtmail !
I tried to follow the documentation and still no extra tags appear in my results. I did:

```
java -jar photon-0.3.5.jar nominatim-import -host localhost -port 5432 -database nominatim -user myuser -password mypassword -languages en
```
I am quite sure that my Nominatim database has `extratags` import style. Then once Nominatim data has been imported, I started Photon with:
```
java -jar photon-0.3.5.jar -cors-any -extra-tags website,phone,opening_hours,cuisine
```
But in my query results I don't need any extra tags (while I was expecting them).

May I know which step I'm missing here?

Thanks 🙏

**Reply by lonvia** [reply]
The -extra-tags option needs to be used during import.

**Reply by xinyuluo** [reply]
@lonvia i see. Yeah it works. Thank you.
